src/sass/styles.scss

```
@@ -259,3 +259,7 @@ $border-thin: 1px solid $color-mid-light !default;
 .no-border-top {
   border-top: none !important;
 }
+
+.text-area {
+  overflow: hidden;
```

If we do this, it probably makes sense to also disable the `resize` - otherwise the user can still resize the textarea with the bottom-right handle, and that would result in a broken UI since the content is not scrollable.
```suggestion
  overflow: hidden;
  resize: none;
```